packages/commonwealth/client/scripts/models/IChainAdapter.ts

```diff
@@ -66,50 +67,41 @@ abstract class IChainAdapter<C extends Coin, A extends Account> {
     }
 
     const {
-      pinnedThreads,
       admins,
-      activeUsers,
+      // activeUsers,
```

Comment:
we are not using `activeUsers`, and this can be removed from `/bulkOffChain` API response